lib/modules/manager/git-submodules/extract.spec.ts

```diff
@@ -1,69 +1,87 @@
 import is from '@sindresorhus/is';
 import { mock } from 'jest-mock-extended';
-import _simpleGit, {
-  Response,
-  SimpleGit,
-  SimpleGitFactory,
-  TaskOptions,
-} from 'simple-git';
+import { Response, SimpleGit, SimpleGitFactory, simpleGit } from 'simple-git';
 import { GlobalConfig } from '../../../config/global';
 import * as hostRules from '../../../util/host-rules';
-import type { PackageFileContent } from '../types';
 import { extractPackageFile } from '.';
 
 jest.mock('simple-git');
-const simpleGit: jest.Mock<Partial<SimpleGit>> = _simpleGit as never;
+const simpleGitFactoryMock = simpleGit as jest.Mock;
```

Comment:
use proper Mock parameter type

Reply:
I got it working with proper types, not sure if that is the best way. If you have any better/cleaner way, happy about feedback :)